File: LLM-Translations/GPT-4/codenet/Python/C/s238775098.c
#include <stdio.h>

int main() {
    int n;
    float res = 0;
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        float x;
        char u[4];
        scanf("%f %s", &x, u);

        if (strcmp(u, "BTC") == 0) {
            res += x * 380000;
        } else {
            res += x;
        }
    }

    printf("%.2f\n", res);
    return 0;
}
// 

File: LLM-Translations/GPT-4/codenet/Python/Java/s238775098.java
import java.util.Scanner;

public class s238775098 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        double res = 0;
        for (int i = 0; i < n; i++) {
            double x = sc.nextDouble();
            String u = sc.next();
            if (u.equals("BTC")) {
                res += x * 380000;
            } else {
                res += x;
            }
        }
        System.out.println(res);
    }
}
// 
